templates/artist/includes/artist_detail_investors.html

```diff
@@ -1,24 +1,91 @@
+{% load humanize %} 
+{% load perdiem %}
+{% load staticfiles %}
+
 <div class="tabs-panel" id="investors">
-    <h2>INVESTORS</h2>
+    {% if user.is_authenticated %}
```

Comment:
Wan't 100% sure the best way to indent the {%  %} tags. Do they act the same way as <div>  or do they get indented no matter how they are used?